tox.ini

```diff
@@ -4,9 +4,19 @@ envlist = py37,flake8
 [testenv]
 deps =
     -rrequirements.txt
+    pytest-cov
 setenv =
  PYTHONPATH=.
```

Comment:
Do we still need to keep this `PYTHONPATH` hack there? It shouldn't be there in the first place and I see you changed the way the unittests are wired a bit.

Reply:
Yeah, it looks like if we drop this, `constants.TEMPLATE_DEPLOYMENT_DIR` ends up having the value of `.tox/py37/lib/python3.7/site-packages/templates/ocs-deployment` instead of `templates/ocs-deployment`, which doesn't work.

I believe this may signal a larger issue with packaging; I've also noticed that all of our top-level modules end up in `$venv/lib/python3.7/site-packages`, which seems wrong...